src/commcare_cloud/ansible/roles/backups/tasks/main.yml

```diff
@@ -55,7 +55,22 @@
     weekday: "1,2,3,4,5,6"
     user: "{{ blobdb_user }}"
     cron_file: backup_blobdb
-  when: "backup_blobdb and 'shared_dir_host' in group_names"
+    state: "{{ 'present' if backup_blobdb and 'shared_dir_host' in group_names else 'absent' }}"
+  tags:
+    - cron
+    - backups
+
+- name: Create Hourly Cron job
+  become: yes
+  cron:
+    name: "Backup blobdb hourly"
+    job: "/usr/local/sbin/create_blobdb_backup.sh hourly {{ blobdb_backup_hours * 60 }}"
+    minute: 15
+    hour: "*/{{ nadir_hour_frequency|default(3) }}"
+    weekday: "1,2,3,4,5,6"
```

Comment:
I see the daily task also does it this way, but is there a reason we aren't including Sunday?